aws/eks/dashboards.tf

```
@@ -95,13 +95,33 @@ resource "aws_cloudwatch_dashboard" "notify_system" {
                 "region": "${var.region}",
                 "period": 60,
                 "stat": "Sum",
-                "title": "SMS Send Rate Per Minute",
+                "title": "SNS Send Rate Per Minute",
```

SMS might be better, no? SNS also sends other types of notifications that are not SMS, unless the metric we use encapsulate all?
```suggestion
                "title": "SNS/SMS Send Rate Per Minute",
```